Android/BluetoothChat/app/src/main/java/ditto/live/bluetoothchat/BluetoothConstants.kt

```diff
@@ -0,0 +1,11 @@
+package ditto.live.bluetoothchat
+import java.util.*
+
+//App service UUID
+val chatDiscoveryServiceID = UUID.fromString("42332fe8-9915-11ea-bb37-0242ac130002")
+
+//Gatt service message characteric ID used
+val chatMessageCharacteristicID = UUID.fromString("43eb0d29-4188-4c84-b1e8-73231e02af95")
+
+//UUID to confirm device connection
+val confirmCharacteristicID = UUID.fromString("f0ab5a15-b003-4653-a248-73fd504c128f")
```

Comment:
While the values are the same, the names and comments of these UUIDs should match what's in the iOS app.

https://github.com/getditto/bluetooth-blog-tutorial/blob/d6354958b2e786b7b2a9d5812669355b102c5c5a/iOS/App/BluetoothChat/Constants/BluetoothConstants.swift#L12-L26

Specifically, the second UUID is used as a service ID rather than a characteristic; the third UUID is a a chat characteristic where messages are written, rather than playing any sort of "confirmation" role.